dpnp/dpnp_container.py

```diff
@@ -111,3 +112,28 @@ def empty(shape,
                           usm_type=usm_type,
                           sycl_queue=sycl_queue_normalized)
     return dpnp_array(array_obj.shape, buffer=array_obj, order=order)
+
+
+def full(shape,
+         fill_value,
+         *,
+         dtype=None,
+         order="C",
+         device=None,
+         usm_type=None,
+         sycl_queue=None):
+    """Validate input parameters before passing them into `dpctl.tensor` module"""
+    dpu.validate_usm_type(usm_type, allow_none=True)
+    sycl_queue_normalized = dpnp.get_normalized_queue_device(fill_value, sycl_queue=sycl_queue, device=device)
+
+    if isinstance(fill_value, dpnp_array):
+        fill_value = fill_value.get_array()
+
+    """Creates `dpnp_array` where every element is equal to fill_value."""
```

Comment:
This docstring is blatantly wrong when `fill_value` is an array. NumPy says "Return a new array of given shape and type, filled with `fill_value`."